tester.py

```diff
@@ -167,6 +176,10 @@ def try_to_alter_key(key, usr_can, admin_can, value = "Dummy", op = '$set'):
     #all the test cases. Test name maps to the update and whether the user and admin
     #should be able to perform the update.
     try_to_fuck_with = {
+        "auth_user": try_to_alter_key("auth",True,False),
+        "auth_admin": try_to_alter_key("auth",False,True),
+        "auth_general1": try_to_alter_key("auth",True,True),
+        "auth_general2": try_to_alter_key("auth",False,False)
```

Comment:
The way this works is that it looks at who an do the updates. So `try_to_alter_key("auth", False, False)` is the only correct thing to do. Everything else will fail.

Also, this file is informally deprecated - I'll work it into normal tests later on.

Reply:
You forgot a comma here, that's why the build is failing right now.